package.json

```diff
@@ -7,7 +7,8 @@
     "dev": "rollup -c -w",
     "start": "sirv public --no-clear",
     "check": "svelte-check --tsconfig ./tsconfig.json",
-    "test": "jest"
+    "test": "jest",
+    "prepare": "husky install"
```

Comment:
I don't agree with having this as a command. Wouldn't you get `husky` by doing the usual `npm install`?

Reply:
This command came after running npx husky-init command